packages/fuel-gauge/src/e2e-script.test.ts

```diff
@@ -12,19 +29,22 @@ describe('Live Script Test', () => {
   const UPGRADE_TX_ID = '0xe2c03044fe708e9b112027881baf9f892e6b64a630a629998922c1cab918c094';
   const UPLOAD_TX_ID = '0x94bc2a189b8211796c8fe5b9c6b67624fe97d2007e104bf1b30739944f43bd73';
 
+  const selectedNetwork: keyof typeof networks = 'devnet';
```

Comment:
Should we use an array for this? Maybe an enum as well?

```ts
export enum NETWORKS {
  DEVNET = 'devnet',
  TESTNET = 'devnet',
}

const selectedNetworks: NETWORKS[] = [NETWORKS.DEVNET, NETWORKS.TESTNET];
```

With this, we can run e2e's selectively as we go, or even target both networks simultaneously.

Makes sense?

Reply:
We should also define these TX IDs inside the Network object so they can be individualized per chain:

 - `MINT_TX_ID`
 - `UPGRADE_TX_ID`
 - `UPLOAD_TX_ID`

Reply:
@Torres-ssf Should we use the opportunity to grab some new IDs from `testnet`?

This could solve #2447 before achieving a final solution via #2471 and/or #2474.

Reply:
@arboleya @petertonysmith94 Getting IDs for `upgrade` and `upload` TXs will be only possible if any of these transactions were already submitted on the `testnet`.